.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+## fixes KILTProtocol/ticket#312
+Please include a summary of the changes provided with this pull request and which issue has been fixed.
+Please also provide some context if necessary.
+
+## How to test:
+
+```
+cd tests
+yarn ts-node 1_identity.ts
+yarn ts-node 2_ctypeFromSchema.ts
+yarn ts-node 3_claim.ts
+yarn ts-node 4_attestation.ts
+yarn ts-node 5_verification.ts
+yarn ts-node 6_verification-with-nonce.ts
+```
+
+## Checklist:
+
+- [ ] The tests and the guide inside the MD are in sync
```

Comment:
```suggestion
- [ ] The tests inside the MD are in sync
- [ ] The guide inside the MD are in sync
```

Reply:
sounds like the tests are inside the MD? Maybe that's even clearer?

```
- [ ] The tests are in sync with the MD code examples
- [ ] The guide inside the MD are in sync with the code examples
```